translations/base-fr.yaml

```diff
@@ -715,14 +715,14 @@ storyRewards:
             connecter un signal booléen (1 / 0) pour l’activer ou le désactiver
             complètement.
     reward_display:
-        title: Afficheur
-        desc: "Vous avez débloqué l’<strong>afficheur</strong> ! Connectez-y un signal
+        title: Affichage
```

Comment:
**Afficheur** was correct

Reply:
I think affichage sounds better, but both work. What do you think Krop?

Reply:
Also, thanks for proofreading!

Reply:
Nope, `Affichage` has a completely different definition. AFAICS, these strings are about an electronic device to display information, hence `afficheur`. 
Quoting Wikipedia[1]:
« en électronique, un afficheur numérique (ou digital) est un dispositif comportant un ou plusieurs emplacements où l'on peut représenter un chiffre, ou une lettre alphabétique. »

vs. affichage[2]:
« technique de communication en extérieur qui consiste à installer et parfois gérer dans un espace public du mobilier urbain destiné à recevoir un support de nature promotionnelle. »

[1] https://fr.wikipedia.org/wiki/Afficheur
[2] https://fr.wikipedia.org/wiki/Affichage

Reply:
ok, bah je remet afficheur du coup